jobs/payment-jobs/tasks/cfs_create_invoice_task.py

```diff
@@ -105,13 +105,8 @@ def _cancel_rs_invoices(cls):
                     for receipt in receipts:
                         CFSService.unapply_receipt(cfs_account, receipt.receipt_number,
                                                    invoice_reference.invoice_number)
-
-                    # Adjust to zero: -invoice.total + invoice.total = 0
-                    adjustment_negative_amount = -invoice.total
-                    CFSService.adjust_invoice(cfs_account=cfs_account,
-                                              inv_number=invoice_reference.invoice_number,
-                                              amount=adjustment_negative_amount)
-
+                    # This used to be adjust invoice, but the suggested way from Tara is to use reverse invoice.
+                    CFSService.reverse_invoice(invoice_reference.invoice_number)
```

Comment:
I tried this by hand on TEST, seems to work.. it does return

```
{
    "credit_memo": "11133",
    "CAS-Returned-Messages": "SUCCESS"
}
```

We don't typically do anything with the credit memo return though.